testplan/web_ui/testing/src/state/appSlice.js

```diff
@@ -0,0 +1,85 @@
+// @ts-nocheck
+import { createSlice } from '@reduxjs/toolkit/dist/redux-toolkit.esm';
+import { headers as DefaultHeaders } from 'axios/lib/defaults';
+import { flattened } from '../Common/utils';
+
+const __DEV__ = process.env.NODE_ENV !== 'production';
+const API_BASE_URL = process.env.REACT_APP_API_BASE_URL;
+
+/** always false in production */
+const boolIfDev = val => __DEV__ ? !!val : false;
```

Comment:
Name this falseIfProd will make the code easier to read, but i wish we don't need this at all.